hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -601,6 +609,12 @@ cdef class PureMarketMakingStrategy(StrategyBase):
         place_bid_price = round(Decimal(place_bid_price)/price_quant) * price_quant
         place_ask_price = round(Decimal(place_ask_price)/price_quant) * price_quant
 
+        if str(maker_name) in self._radar_relay_type_exchanges:
```

Comment:
Again, there should be no `str()` here since you already have a str.

Reply:
done